### PR TITLE
Added support for dropwizard 1.X

### PR DESCRIPTION
**Pull request by vipul0092**

### CODE REVIEW

File: src/main/java/com/cvent/dropwizard/mybatis/datasource/ConfigurableLazyDataSourceFactory.java [hunk 1]
```diff
@@ -17,8 +17,7 @@
 
     /** {@inheritDoc} */
     @Override
-    public ManagedDataSource build(MetricRegistry metricRegistry, String name) throws
-            ClassNotFoundException {
+    public ManagedDataSource build(MetricRegistry metricRegistry, String name) {
```
Comment (vipul0092) [inline src/main/java/com/cvent/dropwizard/mybatis/datasource/ConfigurableLazyDataSourceFactory.java]: change in contract

File: src/main/java/com/cvent/dropwizard/mybatis/sessionbuilder/DefaultSqlSessionFactoryProvider.java [hunk 2]
```diff
@@ -197,7 +197,7 @@ private SqlSessionFactory buildSessionFactory(ConfigurableLazyDataSourceFactory
         try {
             ManagedDataSource ds = dataSource.build(dropwizardEnvironment.metrics(), dataSourceName);
             sessionFactory = new MyBatisFactory().build(dropwizardEnvironment, dataSource, ds, dataSourceName);
-        } catch (ClassNotFoundException e) {
+        } catch (Exception e) {
```
Comment (buerkle) [inline src/main/java/com/cvent/dropwizard/mybatis/sessionbuilder/DefaultSqlSessionFactoryProvider.java]: try/catch can be removed. Neither line has a throws Exception in which case we'll end up wrapping a RuntimeException in another RuntimeException.

**Comment by brentryan** [comment]
Lets update cvent-parent to latest 2.0.1 version as well here.

File: src/main/java/com/cvent/dropwizard/mybatis/sessionbuilder/DefaultSqlSessionFactoryProvider.java (hunk 2 above)
Comment (vipul0092) [reply src/main/java/com/cvent/dropwizard/mybatis/sessionbuilder/DefaultSqlSessionFactoryProvider.java]: Removed

**Pull request merged**